x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/inference/chunking/EmbeddingRequestChunkerTests.java

```diff
@@ -44,7 +44,9 @@ public class EmbeddingRequestChunkerTests extends ESTestCase {
     private static final int MAX_BATCH_SIZE = 512;
 
     public void testEmptyInput_WordChunker() {
-        var batches = new EmbeddingRequestChunker<>(List.of(), 100, 100, 10).batchRequestsWithListeners(testListener());
+        var batches = new EmbeddingRequestChunker<>(List.of(), 100, new WordBoundaryChunkingSettings(100, 10)).batchRequestsWithListeners(
```

Comment:
Do we have a test that uses the `ChunkingSettingsBuilder.DEFAULT_SETTINGS` if we aren't passing one in?

Reply:
We didn't, I added one